### ISSUE TITLE
1.6-beta.29 not work on Windows

### ISSUE THREAD

**Issue by lhz516** [meteor/meteor#9106]
### Environment

Win 10 64x

### Console output

```
=> Started proxy.
=> Errors prevented startup:

   While loading plugin `compileStaticHtmlBatch` from package
   `static-html`:
   C:\tools\isobuild\bundler.js:1894:28: Cannot find module
   "fibers\\future"
   at Object.require (C:\tools\isobuild\bundler.js:1894:28)
   at packages/meteor.js:663:18
   at packages/meteor.js:846:4
   at packages/meteor.js:1380:3


   While loading plugin `minifyStdCSS` from package
   `standard-minifier-css`:
   C:\tools\isobuild\bundler.js:1894:28: Cannot find module
   "fibers\\future"
   at Object.require (C:\tools\isobuild\bundler.js:1894:28)
   at packages/meteor.js:663:18
   at packages/meteor.js:846:4
   at packages/meteor.js:1380:3


=> Your application has errors. Waiting for file change.
```

### Packages
```
meteor-base@1.2.0-beta.29             # Packages every Meteor app needs to have
mobile-experience@1.0.5-beta.29       # Packages for a great mobile UX
mongo@1.3.0-beta.29                   # The database Meteor supports right now
static-html
tracker@1.1.3                 # Meteor's client-side reactive programming library

standard-minifier-css@1.3.4   # CSS minifier run for production mode
standard-minifier-js@2.2.0-beta.29    # JS minifier run for production mode
es5-shim@4.6.15                # ECMAScript 5 compatibility for older browsers
ecmascript@0.9.0-beta.29              # Enable ECMAScript2015+ syntax in app code
shell-server@0.3.0-beta.29            # Server-side component of the `meteor shell` command
dynamic-import@0.2.0-beta.29
server-render@0.2.0-beta.29
```

**Comment by benjamn** [comment]
We're aware of this. Try `meteor update 1.6-beta.30`?

**Comment by lhz516** [comment]
@benjamn I tried `meteor update --release 1.6-beta.30`

Output:
```
This project uses Meteor 1.6-beta.30, which isn't available on Windows. To work with this app on all supported platforms, use meteor update --release METEOR@1.5.2 to pin this app to the
newest Windows-compatible release.
```

**Comment by benjamn** [comment]
Not sure when you tried it, exactly, but it's definitely published now. Please keep trying; this specific problem is fixed in `1.6-beta.30`!

**Comment by lhz516** [comment]
@benjamn I tried it again and it works. Thanks!
